modules/UI/videolayout/LargeVideoManager.js

```diff
@@ -283,9 +286,15 @@ export default class LargeVideoManager {
             }
 
             const isAudioOnly = APP.conference.isAudioOnly();
+
+            // For Plan B enbdpoints with multi-stream support, display an avatar for the original thumbnail.
```

Comment:
Actually I have succeeded to send multiple track on Plan-B by preserving the desktop msid (aka screenshare) and add it to as additional ssrc in video section in Plan-B SDP. We are using multiple stream in this solution which is used by country-wide. In this solution stream does not have simulcast support yet. We did not need also due to the low bandwith consumption (~500kb/s).

Here is the main part of my changes which is done locally. 

`https://github.com/jitsi/jitsi-videobridge/pull/1842#discussion_r839610691`

Could you please consider showing the remote screenshare as another ssrc (stream source) at least. 

In my scenario both endpoint use Plan-B. If one of them uses Unified-Plan we may use this logic by inserting it into the sdp-interop module or library.

I have seen this commit and tested. It did not work on mobile which uses Plan-B. 

`https://github.com/jitsi/sdp-interop/commit/3d49eb4aa26863a3f8d32d7581cdb4321244266b`

Reply:
> Actually I have succeeded to send multiple track on Plan-B by preserving the desktop msid (aka screenshare) and add it to as additional ssrc in video section in Plan-B SDP. We are using multiple stream in this solution which is used by country-wide. In this solution stream does not have simulcast support yet. We did not need also due to the low bandwith consumption (~500kb/s).

Are you talking about the p2p case ? Yes it is possible to send multiple tracks in plan-b by adding both the ssrcs in the same m-line. However, in the bridge case there are some hacks that we do for plan-b that make signaling multiple tracks in plan-b more complicated than the unified plan. Since plan-b is deprecated already and will be removed in M102 (even with origin trial), we did not think that the extra time, effort and complexity involved in making multi-stream work for plan-b was justified.  This is not something we intend to support.

> 
> Here is the main part of my changes which is done locally.
> 
> [inserting desktop ssrc into video mline](https://github.com/jitsi/jitsi-videobridge/pull/1842#discussion_r839610691)
> 
> Could you please consider showing the remote screenshare as another ssrc (stream source) at least.

I do not understand what you mean by this. With multi-stream mode on, remote screenshare is a different source and has an ssrc different from the camera stream.

> 
> In my scenario both endpoint use Plan-B. If one of them uses Unified-Plan we may use this logic by inserting it into the sdp-interop module or library.
> 
> I have seen this commit and tested. It did not work on mobile which uses Plan-B.
> 
> [commit for to PlanB in sdp-interop](https://github.com/jitsi/sdp-interop/commit/3d49eb4aa26863a3f8d32d7581cdb4321244266b)

Again, I don't understand what you mean here, that commit was needed for unified plan clients to extract the msids for the video streams  from the SDP and correctly signal  multiple local video streams to jicofo.

Reply:
> > Anyway, what do you think about mobile app, if you merge this pull request. I could not succeeded to get screenshare track on mobile side.
> 
> I do not understand how this particular commit blocks the screenshare track from being shown on mobile side ? The change you are referring to is modules/UI/videolayout/LargeVideoManager.js which is used only by web.

Sorry for misunderstanding. I did not mention this commit only. I wrote thoose because of plan-B multi-stream support. 

I can see an avatar on mobile, but no track. I have tested on jitsi-meet 7210 version.